docs/src/main/sphinx/security/inspect-jks.rst

```diff
@@ -0,0 +1,148 @@
+.. _jks_certs:
+
+=========
+JKS files
+=========
+
+This topic describes the :ref:`Java KeyStore (JKS) <glossJKS>` certificate
+container formats accepted by Trino to validate TLS/HTTPS client access to the
+cluster.
+
+.. _server_java_keystore:
+
+Java keystore
+-------------
+
+.. note::
+
+  See :ref:`Inspect PEM files <pem_certs>` for information on the preferred
+  certificate format.
+
+The Java KeyStore (JKS) system is provided as part of your Java installation.
+Private keys and certificates for your server are stored in a *keystore* file.
+The keystore file is always password-protected.
+
+To be recognized as valid by the Trino :doc:`CLI </installation/cli>` and modern
```

Comment:
I would say Trino clients here.  I would also say "automatically recognized", you can always configure clients

Reply:
done